client/src/components/feddback-comment.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap'
+import '../style/feedback.css'
+
+class FeedbackComment extends Component {
+
+
+    render() {
+
+        return (
+
+            <Container className="feedbackUserContainer">
+                <img src="/img/defAvatar.png" width="50px" height="50px" alt="defAvava"></img>
```

Comment:
If I were you, I'd import image in import section and use here it inside `<img>` block. As for styles, I'd create another `.css` file referring to this image style. It will be more readable and simple to do like this.

Reply:
See also lines 24, 28, 44

Reply:
we have folder img , and for using img we print srs="your_folder_img/name.png"

Reply:
It is good since it works. My offer is just for readability :)